### ISSUE TITLE
GridMapping Unit Test fails

### ISSUE THREAD

**Issue by TonioF** [dcs4cop/xcube#598]
One unit test fails. This has first been observed after CI build 1133.
```
FAILED test/core/gridmapping/test_base.py::GridMappingTest::test_transform - 
AssertionError: Tuples differ: (400, 200) != (360, 180)
```

 - [Last successful build: 1132](https://ci.appveyor.com/project/bcdev/xcube/builds/42378793)
 - [First failing build: 1133](https://ci.appveyor.com/project/bcdev/xcube/builds/42384450)

**Comment by TonioF** [comment]
Issue did not occur due to changes in xcube code, but due to the release of xarray 0.21

**Comment by TonioF** [comment]
Closed by #599
